trace/trace_test.go

```diff
@@ -230,6 +230,32 @@ func TestStartSpanWithRemoteParent(t *testing.T) {
 	}
 }
 
+func TestLocalRootSpanID(t *testing.T) {
+	ctx, span1 := StartSpan(context.Background(), "span1")
+	if span1.localRootSpanID == (SpanID{}) {
+		t.Errorf("exporting root span: expected nonzero localRootSpanID")
+	}
+
+	_, span2 := StartSpan(ctx, "span2")
+	if err := checkChild(span1.spanContext, span2); err != nil {
+		t.Error(err)
+	}
+	if got, want := span2.localRootSpanID, span2.localRootSpanID; got != want {
```

Comment:
want should be span1.localRootSpanID.

Reply:
Thanks - fixed. I rushed the tests a little bit.

(The tests aren't too important though, I mostly just put this up for comments on the approach.)